applications/nrf_desktop/doc/dvfs.rst

```diff
@@ -0,0 +1,116 @@
+.. _nrf_desktop_dvfs:
+
+DVFS module
+###########
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF Desktop Dynamic Voltage and Frequency Scaling (DVFS) module is responsible for switching the frequency and voltage according to the application needs.
+The module chooses the highest frequency required by the application states and selects voltage accordingly.
```

Comment:
To be precise frequency is also controlled under the hood. We have points(freq, V), which are ordered - the way we call them doesn't matter. We could name them a,b,c,... and let user select one. The way you try to present this shows false impression that user have possibility to choose any frequency. We call them frequency because for most people it is the most important part.

Reply:
You set frequency through the API (passing it explicitly as a parameter). The frequency rounding and voltage update is done under the hood by the underlying drivers' implementations.